src/main/resources/config/liquibase/changelog/20191210131150_changelog.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet author="hashemy" id="20191210131150">
+        <createTable tableName="grading_instruction">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="credits" type="DOUBLE"/>
+            <column name="level" type="VARCHAR(255)"/>
```

Comment:
I don't know the concept behind the `level` column, but we can consider it being an ENUM right? Or does every Instructor/TA define their own levels?

Reply:
Good point! I think we need to discuss this further and look at the advantages and disadvantages of giving instructors the freedom to define their own grading scale.

Reply:
We can discuss it on Tuesday but I think the way you meant it to be is perfectly okay - “poor”, “average”, “good” or another more detailed, but still predefined, list makes more sense in my opinion as otherwise it could get really inconsistent